### PR TITLE
BATPT-1195: Move the Existing Knightlight server into the Mono-Repo

### PR DESCRIPTION
**Pull request by czingrafRevacomm**

### CODE REVIEW

File: java-server/Dockerfile
Comment (BacchusJackson) [inline java-server/Dockerfile:1]: Ignoring now, but this should be updated to use a user so it's not running as root

File: java-server/settings.xml
Comment (BacchusJackson) [inline java-server/settings.xml:1]: Should remove the BC_QMMS reference

Comment (czingrafRevacomm) [reply java-server/settings.xml:1]: deleting the settings.xml completely since it's not needed

File: java-server/Dockerfile
Comment (czingrafRevacomm) [reply java-server/Dockerfile:1]: Will work on the dockerfile to use a user

Comment (czingrafRevacomm) [reply java-server/Dockerfile:1]: user is added for dockerfile for running

**Review by BacchusJackson — APPROVED** [review]

**Pull request merged**